Makefile

```diff
@@ -241,6 +243,82 @@ catalog-deploy:
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml	 | kubectl create -f -  || true
+	kubectl wait --timeout=120s --for=condition=ready pod -l app=loki
+	-pkill --oldest --full "3100:3100"
+	kubectl port-forward --address 0.0.0.0 svc/loki 3100:3100 2>&1 >/dev/null &
+	@echo -e "\n===> loki endpoint is available on http://localhost:3100\n"
+
+.PHONY: undeploy-loki
+undeploy-loki: ## Undeploy loki.
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml	 | kubectl --ignore-not-found=true  delete -f -  || true
+	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl --ignore-not-found=true  delete -f -  || true
+	-pkill --oldest --full "3100:3100"
+
+.PHONY: deploy-grafana
+deploy-grafana: ## Deploy grafana.
+	@echo "### Deploying grafana"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	./hack/deploy-grafana.sh $(NAMESPACE)
+	-pkill --oldest --full "3000:3000"
+	kubectl port-forward --address 0.0.0.0 svc/grafana 3000:3000 2>&1 >/dev/null &
+	@echo -e "\ngrafana ui is available (user: admin password: admin) on http://localhost:3000\n"
+
+.PHONY: undeploy-grafana
+undeploy-grafana: ## Undeploy grafana.
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+	kubectl delete --ignore-not-found=true route grafana
+	kubectl delete --ignore-not-found=true service grafana
+	kubectl delete --ignore-not-found=true deployment grafana
+	kubectl delete --ignore-not-found=true configMap grafana-datasources
+	-pkill --oldest --full "3000:3000"
+
+.PHONY: ocp-deploy
+ocp-deploy: manifests generate fmt lint deploy-loki deploy-grafana install deploy-sample-cr   ## OCP deploy (loki, grafana and example-cr)
+	oc expose service grafana || true
+	@grafana_url=$$(oc get route grafana -o jsonpath='{.spec.host}'); \
+	echo -e "\nAccess grafana on OCP using: http://"$$grafana_url"\n"
+	oc expose service loki || true
+	@loki_url=$$(oc get route loki -o jsonpath='{.spec.host}'); \
+	echo -e "\nAccess loki on OCP using: http://"$$loki_url"\n"
+
+.PHONY: ocp-undeploy
+ocp-cleanup: undeploy-loki undeploy-grafana uninstall undeploy-sample-cr   ## OCP cleanup
+	-PID=$$(pgrep --oldest --full "main.go"); pkill -P $$PID; pkill $$PID
+
+.PHONY: ocp-run
+ocp-run: ocp-cleanup ocp-deploy   ## OCP-deploy + run the operator locally
+	@echo "### Running operator locally (in background process)"
+	-PID=$$(pgrep --oldest --full "main.go"); pkill -P $$PID; pkill $$PID
+	go run ./main.go &
+	@echo "### Waiting for goflow-kube pod to be ready"
+	while : ; do kubectl get deployment goflow-kube && break; sleep 1; done
+	kubectl wait --timeout=120s --for=condition=ready pod -l app=goflow-kube
```

Comment:
@stleerh 

This was an hour of frustration for me :-) 

314 is waiting for the **deployment** to appear .... and 315 is waiting for the **pod** to be ready using the kubectl  `wait` functionality  ... 
We need 314 because without that the deployment doesn't exist so we can't even ask for the pods' condition ... and we get an error immediately 


I LOVE K8S :-)